releasenotes/notes/apm-fix-sql-obfuscation-bind-vars-with-digit-prefix-ee2792fc124adfd8.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fix SQL obfuscation error on statements using bind variables starting with digits
```

This package is technically not fully APM anymore, but perhaps we should continue the previously used convention?
```suggestion
    APM: Fix SQL obfuscation error on statements using bind variables starting with digits
```

---

That works for me! Committed the change. Thanks, @gbbr !